src/frameworks/CoreBluetooth/include/CoreBluetooth/CBCentralManagerConstants.h

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Don't forget to include this file in [CoreBluetooth.h](https://github.com/jlucfarias/darling/blob/add-symbols-vscode/src/frameworks/CoreBluetooth/include/CoreBluetooth/CoreBluetooth.h)